.circleci/config.yml

```
@@ -3,7 +3,7 @@ version: 2.1
 parameters:
   ci_builder_image:
     type: string
-    default: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:v0.43.0
+    default: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:v0.46.0-rc.4
```

Is this just temporary pending an updated 0.46.0 ci builder image release?

---

@sebastianst what was the idea here?

---

I just finalized the `ci-builder/v0.46.0` release, you can update this here to use it.
```suggestion
    default: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:v0.46.0
```

---

committed this myself

---

I should have waited for the docker image to be built 😬

---

@sebastianst The new image is having that SIGILL issue again. Is it using the same foundry version?

---

yeah I finalized rc.4 but then didn't update the versions.json. I now did another update round of foundry to latest nightly with `ci-builder/v0.46.1`. checking now if this fixes it.

---

I think that finally worked! Just one more flake to pass